apps/data/database/initdb.sql

```diff
@@ -0,0 +1,68 @@
+CREATE TYPE STATUS_TYPE AS ENUM ('AVAILABLE', 'SOLD');
+
+CREATE TYPE LOCATION_TYPE AS (
+    latitude float,
+    longitude float
+);
+
+-- Creating Tables:
+
+CREATE TABLE UserTable (
+    user_id SERIAL PRIMARY KEY,
+    username VARCHAR NOT NULL,
+    email VARCHAR NOT NULL,
+    password VARCHAR NOT NULL,
+    name VARCHAR,
+    bio TEXT,
+    profile_pic_url VARCHAR,
+    joining_date TIMESTAMP DEFAULT NOW(),
```

Comment:
if we specified joining_date in the api spec we can leave it as that, but otherwise this sould probably be renamed to createdAt. Also will need a verified field as was just discussed in the meeting

Reply:
or date_created to line up with the other tables